app/src/main/java/com/anzelmas/quizzando/QuizQuestionActivity.kt

```diff
@@ -14,6 +15,145 @@ import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
```

Comment:
```
import androidx.databinding.DataBindingUtil
import androidx.databinding.ViewDataBinding
```

Those imports are not in use, easy way to get rid of those - use shortcuts: https://stackoverflow.com/questions/43425945/remove-unused-import-in-reformat-code-in-android-studio/43426621